---
title: "Diauxic shift of *Roseburia intestinalis*"
subtitle: "Enables slow growth after depletion of glucose"
format: 
  revealjs:
    embed-resources: true
    slide-number: true
editor: visual
---

## *R. intestinalis* Adapts via Diauxic Shift

::::: {.columns}

::: {.column width="60%"}
### Mechanism of Survival
* **Slow-Growth Mode:** *R. intestinalis* survives depletion of glucose and mucin products by drastically reducing growth rate.
* **Metabolic Switch:** Metabolomics & RNA-seq confirm a **diauxic shift** toward consuming:
    - **Acetate**
    - **Lactate**

### Gene Expression
Consistent upregulation observed in:
* **Sugar utilization pathways** (glycolysis, pyruvate utilization)
* **Butyrate-producing pathways**
:::

::: {.column width="40%"}
![](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/ismej/17/11/10.1038_s41396-023-01501-1/2/m_41396_2023_1501_fig1_html.jpeg?Expires=1766109298&Signature=ebtPbc~ey20WHF3j3xQO9reMh1rR64ZuSi-Ymts7uYraP8Vim54OMMh4AaRom-xwTEhQtw58LOh6vGoyG0ZWqQerzKEKIEWQ1EQ1we78pqxePO9peb7KxlMNFvvxKcgyD1xzES46ch0tZGluz83kA2lfJd-hJMXMoSJ8JUykiWMm~kHEJc8EgCpz7H6pXEjLleQvKLNSxxTtqnQ3VNIc8utOij15ydx6DW2j9oiVWKlqOYrrtFLv7bghejVX7uW4xHiuuq5gueug-wThPWHwYJsVEdIQmRlAmrxf-1fvmLjpYQsiH98qCk86-ufR4dG542PZ2MNPQY34RxtUE-G6Gw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA){width="100%" fig-align="center"}

<div style="font-size: 0.55em; line-height: 1.2; margin-top: 1em; padding: 0.8em; border-left: 3px solid #4a78d4; background-color: #f8f9fa;">
**Source:** Liu, B., Garza, D.R., Gonze, D. *et al.* (2023). <br>
*Starvation responses impact interaction dynamics of human gut bacteria...* <br>
**ISME Journal** 17. [DOI: 10.1038/...](https://doi.org/10.1038/s41396-023-01501-1)
</div>
:::

:::::


## Data overview {.smaller}

```{r}
ri_metadata<- read.csv("../_raw/01_metadata_data_load.csv")
ri_expression_data <- read.csv("../_raw/01_expression_data_load.csv")
ri_annotation <- read.csv("../_raw/01_annotation_data_load.csv")
ri_fermentation_data <- read.csv("../_raw/01_fermentation_data_load.csv")
```

::: incremental
-   Fermentation

    -   containing `r nrow(ri_fermentation_data)` experiments with data:
    -   `r colnames(ri_fermentation_data)[3:length(ri_fermentation_data)]`

<!-- -->

-   RNA expression data

    -   containing recorded expression of `r nrow(ri_expression_data)` genes in `r ncol(ri_expression_data) -2` experiments

-   Annotation for expression dataset

    -   Coupling internal gene ID's to gene names , KO ID's, and COG ID's

<!-- -->

-   Meta-data for fermentation dataset

    -   Coupling RNA expression samples with fermentation batches and time points
:::

## Data Cleaning and Augmentation

Removing empty time-points from fermentation dataset

```{r}
#| echo: true
#| eval: false
#| code-line-numbers: "1|2|3-4|5"
ri_fermentation_data |>
  filter(Exp == "RI_batch_WC1") |>
  select( "Rowname", (starts_with("T") & ends_with("h")) ) |>
  select(where(~!all(is.na(.x)) ) ) |>
  write_csv("../data/02_fermentation_data_clean.csv")
```

Subsetting RNA expression to wild type R. intestinalis only

```{r}
#| echo: true
#| eval: false
#| code-line-numbers: "1|2-4|5|6|7-10"
ri_expression_data |> 
  pivot_longer(cols = -X,
               
               names_to = "sample",
               values_to = "expression") |> 
  inner_join(ri_metadata, by = c("sample" = "GC.code" )) |> 
  filter(Exp == "RI_batch_WC") |> 
  select(X,expression,sample) |> 
  pivot_wider(names_from = sample,
              values_from = expression) |> 
  write_csv("../data/02_data_clean.csv")
```

## Correlation Matrix from Gene Expression data

::::: columns
::: column
-   Input data was a total of 9 samples across 3 time conditions
-   Matrix reveals co-expression patterns in *R. intestinalis* gene expression data
-   Co-regulated genes can potentially contribute to the same pathways
:::

::: column
![](../results/CoExpression.png){width="100%"}
:::
:::::

## K-Means Identifies 3 Distinct Clusters

:::::: {.columns .v-center-container}
:::: {.column width="40%"}
-   **Methodology:** Applied K-means clustering to the correlation matrix.
-   **Observation:** Genes show clear separation in 2D PCA space.

::: {.callout-note appearance="simple"}
**Note:** The cutoff for 3 clusters was selected for regulation validation purposes.
:::
::::

::: {.column width="60%"}
![](../results/clusters.png){fig-align="center" width="100%"}

<center><small>Figure 1: PCA Projection of Gene Clusters</small></center>
:::
::::::

## Pathway Enrichment Reveals Distinct Cluster Functions

::::: {.columns .v-center-container}
::: {.column width="40%"}
### Observations

-   Major pathways (top bars) are shared across all three clusters.
-   Certain pathways are exclusive to Cluster 1 or 2 (e.g., lower yellow/green bars).
:::

::: {.column width="60%"}
![](../results/topKOcodes.png){fig-align="center" width="100%"}
:::
:::::

## Differential Expression Analysis {.smaller}

::::: {.columns .v-center-container}

::: {.column width="40%"}
* This volcano plot visualizes the transcriptional changes that *R. intestinalis* undergoes during the transition.

* The highly upregulated genes reflect the shift toward alternative energy sources needed after glucose depletion, including pathways for acetate and lactate consumption.

* The highly downregulated genes correspond to pathways that are no longer prioritized once glucose is exhausted, particularly those involved in glucose and pyruvate utilization.
:::

::: {.column width="60%"}
![](../results/volcano.png){width="100%" fig-align="center"}
:::

:::::

## Dynamic Regulation of Key Genes {.smaller}

::::: {.columns .v-center-container}

::: {.column width="40%"}
* **Visualizing Change:** These 6 genes represent the most significant expression changes identified in the RNA-seq analysis across time points (4h, 12h, 48h).
* **Distinct Phases:**
    * **Early Expression (Red/Green):** Genes essential for initial rapid growth.
    * **Late Shifts (Blue):** Genes that spike at 48h likely drive the **diauxic shift** and survival maintenance.

:::

::: {.column width="60%"}
![](../results/topcounts.png){width="100%" fig-align="center"}
:::

:::::

## *R. intestinalis* Exhibits Metabolic Resilience {.smaller}

::::: {.columns}

::: {.column width="50%"}
![](../results/growthCurve.png){height="400" fig-align="center"}

**1. Survival Stability**
Active cell counts remain stable even after nutrient depletion.
:::

::: {.column width="50%"}
![](../results/fullgrowthcurve.png){height="400" fig-align="center"}

**2. Fermentation Dynamics**
Butyrate continues rising as glucose is exhausted.
:::

:::::

## Conclusions

::: {.incremental}
-   **1. Functional Architecture**
    -   Gene co-expression matrix analysis, K-means clustering, and enrichment analysis reveal a structured functional organization within the genome.

-   **2. Metabolic Flexibility**
    -   The **diauxic shift** to acetate and glucose consumption is supported, both by RNA-seq and fermentation data.
    -   This metabolic switch enables a **"slow-growth" maintenance mode**, allowing *R. intestinalis* to maintain cell viability and continue producing butyrate.
