---
title: "Diauxic shift of *Roseburia intestinalis*"
subtitle: "Enables slow growth after depletion of glucose"
format: 
  revealjs:
    embed-resources: true
    slide-number: true
editor: visual
---

## Introduction to Subject

*R. intestinalis* survives post-glucose (and post-mucin-degradation products) depletion by entering a slow-growth mode.

A combination of targeted metabolomics and RNA-seq showed that the slow growth mode of *R. intestinalis* represents a diauxic shift towards acetate and lactate consumption (both leading to butyrate production).

*R. intestinalis* consistently increased the expression of genes involved in glycolysis, pyruvate utilization, mucin hydrolysis, galactose and mannose utilization, and the butyrate-producing process after depletion of glucose/pyruvate.

[Liu, B., Garza, D.R., Gonze, D. et al. Starvation responses impact interaction dynamics of human gut bacteria Bacteroides thetaiotaomicron and Roseburia intestinalis. ISME J 17, 1940â€“1952 (2023). <https://doi.org/10.1038/s41396-023-01501-1>]{style="font-size: 30%;"}

## Data overview {.smaller}

```{r}
ri_metadata<- read.csv("../_raw/01_metadata_data_load.csv")
ri_expression_data <- read.csv("../_raw/01_expression_data_load.csv")
ri_annotation <- read.csv("../_raw/01_annotation_data_load.csv")
ri_fermentation_data <- read.csv("../_raw/01_fermentation_data_load.csv")
```

::: incremental
-   Fermentation

    -   containing `r nrow(ri_fermentation_data)` experiments with data:
    -   `r colnames(ri_fermentation_data)[3:length(ri_fermentation_data)]`

<!-- -->

-   RNA expression data

    -   containing recorded expression of `r nrow(ri_expression_data)` genes in `r ncol(ri_expression_data) -2` experiments

-   Annotation for expression dataset

    -   Coupling internal gene ID's to gene names , KO ID's, and COG ID's

<!-- -->

-   Meta-data for fermentation dataset

    -   Coupling RNA expression samples with fermentation batches and time points
:::

## Data Cleaning and Augmentation

Removing empty time-points from fermentation dataset

```{r}
#| echo: true
#| eval: false
#| code-line-numbers: "1|2|3-4|5"
ri_fermentation_data |>
  filter(Exp == "RI_batch_WC1") |>
  select( "Rowname", (starts_with("T") & ends_with("h")) ) |>
  select(where(~!all(is.na(.x)) ) ) |>
  write_csv("../data/02_fermentation_data_clean.csv")
```

Subsetting RNA expression to wild type R. intestinalis only

```{r}
#| echo: true
#| eval: false
#| code-line-numbers: "1|2-4|5|6|7-10"
ri_expression_data |> 
  pivot_longer(cols = -X,
               
               names_to = "sample",
               values_to = "expression") |> 
  inner_join(ri_metadata, by = c("sample" = "GC.code" )) |> 
  filter(Exp == "RI_batch_WC") |> 
  select(X,expression,sample) |> 
  pivot_wider(names_from = sample,
              values_from = expression) |> 
  write_csv("../data/02_data_clean.csv")
```

## Correlation Matrix from Gene Expression data

::::: columns
::: column
-   Input data was a total of 9 samples across 3 time conditions
-   Matrix reveals co-expression patterns in *R. intestinalis* gene expression data
-   Co-regulated genes can potentially contribute to the same pathways
:::

::: column
![](../results/CoExpression.png){width="100%"}
:::
:::::

## K-Means Identifies 3 Distinct Clusters

:::::: {.columns .v-center-container}
:::: {.column width="40%"}
-   **Methodology:** Applied K-means clustering to the correlation matrix.
-   **Observation:** Genes show clear separation in 2D PCA space.

::: {.callout-note appearance="simple"}
**Note:** The cutoff for 3 clusters was selected for regulation validation purposes.
:::
::::

::: {.column width="60%"}
![](../results/clusters.png){fig-align="center" width="100%"}

<center><small>Figure 1: PCA Projection of Gene Clusters</small></center>
:::
::::::

## Pathway Enrichment Reveals Distinct Cluster Functions

::::: {.columns .v-center-container}
::: {.column width="40%"}
### Observations

-   Major pathways (top bars) are shared across all three clusters.
-   Certain pathways are exclusive to Cluster 1 or 2 (e.g., lower yellow/green bars).
:::

::: {.column width="60%"}
![](../results/topKOcodes.png){fig-align="center" width="100%"}
:::
:::::

## Expression

![](../results/volcano.png)

## Top Counts {.scrollable}

![](../results/topcounts.png)

## Fermentation

::::: columns
::: {.column width="50%"}
![](../results/growthCurve.png)
:::

::: {.column width="50%"}
![](../results/fullgrowthcurve.png)
:::
:::::

::: {style="font-size: 70%;"}
R. intestinalis' active cell count remains stable, even after the nutrients are depleted. This showcases this species' resilience. Butyrate concentration grows as glucose levels are dropping, which is expected, as butyrate is the fermentation product of R. intestinalis. The exact mechanism of its carbon uptake after glucose levels have dropped remains elusive
:::

## Conclusion
