---
title: "Presentation"
format: 
  revealjs:
    embed-resources: true
editor: visual
---

## Introduction to subject

R. intestinalis survives post-glucose (and post-mucin-degradation products) depletion by entering a slow-growth mode.

A combination of targeted metabolomics and RNA-seq showed that the slow growth mode of R. intestinalis represents a diauxic shift towards acetate and lactate consumption (both leading to butyrate production).

R. intestinalis consistently increased the expression of genes involved in glycolysis, pyruvate utilization, mucin hydrolysis, galactose and mannose utilization, and the butyrate-producing process after depletion of glucose/pyruvate.

## Data overview {.smaller}

```{r}
ri_metadata<- read.csv("../_raw/ri_metadata.csv")
ri_expression_data <- read.csv("../_raw/ri_expression_data.csv")
ri_annotation <- read.csv("../_raw/ri_annotation.csv")
ri_fermentation_data <- read.csv("../_raw/ri_fermentation_data.csv")
```

::: incremental
-   Fermentation

    -   containing `r nrow(ri_fermentation_data)` experiments with data:
    -   `r colnames(ri_fermentation_data)[3:length(ri_fermentation_data)]`

<!-- -->

-   RNA expression data

    -   containing recorded expression of `r nrow(ri_expression_data)` genes in `r ncol(ri_expression_data) -2` experiments

-   Annotation for expression dataset

    -   Coupling internal gene ID's to gene names , KO ID's, and COG ID's

<!-- -->

-   Meta-data for fermentation dataset

    -   Coupling RNA expression samples with fermentation batches and time points
:::

## data cleaning and augmentation

Removing empty time-points from fermentation dataset

```{r}
#| echo: true
#| eval: false
#| code-line-numbers: "1|2|3-4|5"
ri_fermentation_data |>
  filter(Exp == "RI_batch_WC1") |>
  select( "Rowname", (starts_with("T") & ends_with("h")) ) |>
  select(where(~!all(is.na(.x)) ) ) |>
  write_csv("../data/02_fermentation_data_clean.csv")
```

Subsetting RNA expression to wild type R. intestinalis only

```{r}
#| echo: true
#| eval: false
#| code-line-numbers: "1|2-4|5|6|7-10"
ri_expression_data |> 
  pivot_longer(cols = -X,
               
               names_to = "sample",
               values_to = "expression") |> 
  inner_join(ri_metadata, by = c("sample" = "GC.code" )) |> 
  filter(Exp == "RI_batch_WC") |> 
  select(X,expression,sample) |> 
  pivot_wider(names_from = sample,
              values_from = expression) |> 
  write_csv("../data/02_data_clean.csv")
```

## analysis results: co-expression

::::: columns
::: column
-   Input data was a total of 9 samples across 3 time conditions (with 3 replicates each)
-   Matrix reveals co-expression patterns in R. Intestinalis gene expression data
-   Co-regulated genes can potentially contribute to the same pathways
:::

::: column
![](../results/CoExpression.png){width="100%"}
:::
:::::

## analysis results: clustering

::::: columns
::: column
-   K-means clustering was applied to the correlation matrix from the previous slide
-   There is a clear separation of genes in the 2D space
-   Genes are grouped into 3 clusters (arbitrary cutoff for regulation validation)
:::

::: column
![](../results/clusters.png){width="100%"}
:::
:::::

## top KO codes per cluster

::::: columns
::: column
-   Annotation of R. Intestinalis genome enables matching of Gene ids with KO terms
-   Mapping of KO terms to pathways enables the calculation of pathways percentages across clusters
-   As expected, some pathways are represented across all clusters, whereas others are limited to 1 or 2 clusters.
:::

::: column
![](../results/topKOcodes.png){width="100%"}
:::
:::::

## differential expression

## fermentation

![](../results/growthCurve.png)

## full fermentation

![](../results/fullGrowthCurve.png)

## conclusion
